cmake/usCTestScript_custom.cmake

```diff
@@ -23,7 +23,7 @@ set(US_TEST_STATIC 1)
 set(US_SOURCE_DIR "${CMAKE_CURRENT_LIST_DIR}/../")
 
 set(US_BUILD_CONFIGURATION )
-foreach(i RANGE 7)
+foreach(i RANGE 3)
```

Comment:
is it possible to not hard code the max range? Could it be derived somehow? I could see how adding/removing build configurations frequently could lead to this value not being updated.